### ISSUE TITLE
"Invalid empty payload" when decoding fragmented binary broadcast message

### ISSUE THREAD

**Issue by gs11** [M0r13n/pyais#157]
Trying to parse the following data picked up by my AIS receiver:
```
!AIVDM,2,1,0,A,8@2R5Ph0GhRbUqe?n>KS?wvlFR06EuOwiOl?wnSwe7wvlOwwsAwwnSGmwvwt,0*4E
!AIVDM,2,2,0,A,,0*16
```

It yields an `pyais.exceptions.InvalidNMEAMessageException` exception while it parses fine in e.g. https://www.aggsoft.com/ais-decoder.htm

It's a bit weird that this message was sent in fragments as this would be the same thing:
```
!AIVDM,1,1,0,A,8@2R5Ph0GhRbUqe?n>KS?wvlFR06EuOwiOl?wnSwe7wvlOwwsAwwnSGmwvwt,0*4E
```

**Comment by gs11** [comment]
Thanks! I'm very shallowly familiar with the codebase but the PR looked good to me

**Comment by M0r13n** [comment]
@gs11 Thanks for you fast feedback! The fixes are available in v2.8.3. :)

Have a great day!
